test/forms/income/AnyOtherIncomeFormProviderSpec.scala

```diff
@@ -14,32 +14,39 @@
  * limitations under the License.
  */
 
-package forms
+package forms.income
 
 import forms.behaviours.BooleanFieldBehaviours
 import play.api.data.FormError
 
 class AnyOtherIncomeFormProviderSpec extends BooleanFieldBehaviours {
 
-  val requiredKey = "anyOtherIncome.error.required"
-  val invalidKey = "error.boolean"
+  ".value" - {
```

Comment:
What is `.value` meaning here?

Reply:
I appreciate it is scaffold-generated tests, but what is it signifying? When do we every call `.value` on our form provider objects?

Reply:
It's the form mapping of (value -> Boolean) but you are right it's pretty redundant in all the form specs. Think maybe should remove it in the CheckYour... -> ...CYA PR though instead

Reply:
It may just be a case of renaming the test string to something more representative of what we are testing for readbility.